eventtracking/processors/filters.py

```diff
@@ -0,0 +1,44 @@
+"""
+Filter configured through admin panel. Filters out the events that
+do not need to be routed
+"""
+import logging
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+logger = logging.getLogger(__name__)
+
+
+class RegExFilterProcessor:
+    """
+    Filter the events using regex. The regex list Can be configured
+    through the Admin panel.
+    """
+
+    def __call__(self, event, **kwargs):
+        # To avoid the "Models aren't loaded yet" error,
+        # importing the model here
+        # pylint: disable=import-outside-toplevel
+        from eventtracking.django.models import RegExFilter
+        try:
+            regex_filter = RegExFilter.objects.all()[0]
+        except IndexError:
+            logger.warning('Regular Expression Filter does not have any configurations. '
+                           'Cannot filter events.')
+            return event
+
+        event_name = event.get('name')
+
+        if not regex_filter.is_enabled:
+            logger.warning(
+                'Regular Expression Filter is not enabled. Allowing the event %s to pass through.',
+                event_name
+            )
+            return event
+
+        if not regex_filter.string_passes_test(event_name):
+            logger.info('Event %s is not allowed to be further processed.', event_name)
+            raise EventEmissionExit
+
+        logger.info('Event %s is allowed to be further processed.', event_name)
```

Comment:
not need this logger since we are already logging in case of event is not allowed to propagate.